bbc/simorgh
# WSTEAM1-926 - Adds PGL CAF rendering support for article page component

Resolves JIRA [WSTEAM1-926](https://jira.dev.bbc.co.uk/browse/WSTEAM1-926)

Overall changes
======
Adds PGL checks for formatting within the Article page when CAF rendererEnv is passed through.

Code changes
======

- Add route check for PGL for CAF
- Adds a single column layout for Article page component when asset type is PGL
- Removes most read section when asset type is PGL

Testing
======
1. Startup Simorgh
2. Visit a PGL asset with CAF query param e.g. http://localhost:7080/pidgin/50913502?renderer_env=caflive
3. Visit a PGL asset without CAF query param e.g. http://localhost:7080/pidgin/50913502?renderer_env=live
4. Check for any differences

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
